addons/web/static/src/views/kanban/kanban_controller.scss

```diff
@@ -511,9 +511,32 @@
             background-color: var(--KanbanColumn__highlight-background) !important;
             box-shadow: -1px 0px 0px 0px var(--KanbanColumn__highlight-border) inset,
                 1px 0px 0px 0px var(--KanbanColumn__highlight-border) inset;
+
+            .o_kanban_header {
+                // hack to display correct background color
+                margin: 0 calc(-1 * var(--KanbanGroup-padding-h) + 1px);
+                padding: 0 calc(var(--KanbanGroup-padding-h) - 1px);
+            }
+        }
+    }
+
+    .o_kanban_group .o_kanban_header,
+    .o_quick_create_folded {
+        position: sticky; // keep it visible
+        z-index: 1;
+        background-color: $o-webclient-background-color;
```

Comment:
this one should be only if the group isn't folded. There's a glitch otherwise 
![mcm](https://user-images.githubusercontent.com/9432572/212275906-dee4c4ff-b06c-41a0-ab4b-90147cd0bb69.png)